demos/JWST/S1_nirx_template.ecf

```diff
@@ -25,6 +25,16 @@ skip_gain_scale     False
 #Pipeline stages parameters
 jump_rejection_threshold  4.0 #float, default is 4.0, CR sigma rejection threshold
 
+#Background subtraction
+grouplevel_bg 		True
+ncpu				8
+bg_y1 				6
+bg_y2 				26
```

Comment:
Also, I assume this is only for NIRSpec and may not work well for NIRCam (and especially not NIRISS)? We'll likely need to break the templates for S1 into NIRCam, NIRSpec, and NIRISS if it does end up being instrument dependent